src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,63 @@
+.forecast-details__wrapper {
+    display: grid;
+    flex-direction: row;
```

Comment:
CSS files should be named using `kebab-case` as the classes themselves are. This is the convention, although some companies will have their own preferences